layouts/compliance/cis.html

```diff
@@ -0,0 +1,233 @@
+                </li>
+
+                <li>
+                    <strong>Key Management Service (KMS):</strong> Use AWS KMS to manage and
+                    rotate encryption keys. Ensure that access to KMS keys is restricted and
+                    monitored.
+                </li>
+            </ul>
+            <h4>5. Auditing and Assessment Tools</h4>
+            <ul>
+                <li>
+                    <strong>AWS Config:</strong> Use AWS Config to assess, audit, and
+                    evaluate the configurations of your AWS resources. Set up rules to
+                    automatically check for compliance with CIS benchmarks and trigger
+                    alerts for non-compliance.
+                </li>
+
+                <li>
+                    <strong>AWS Security Hub:</strong> Enable AWS Security Hub to aggregate
+                    security findings and compare them against CIS Benchmarks. Security Hub
+                    can also automatically generate compliance reports.
+                </li>
+
+                <li>
+                    <strong>CIS-CAT Tool:</strong> Use the CIS Configuration Assessment Tool
+                    (CIS-CAT) to automate the assessment of your AWS environment against the
+                    CIS Benchmarks.
+                </li>
+            </ul>
+            <h4>6. Automation and Continuous Compliance</h4>
+            <ul>
+                <li>
+                    <strong>Infrastructure as Code (IaC):</strong> Use tools like AWS
+                    CloudFormation or Terraform to define your infrastructure in code. This
+                    allows you to apply CIS-compliant configurations consistently and easily
+                    across your environments.
+                </li>
+
+                <li>
+                    <strong>Automated Remediation:</strong> Implement automated remediation
+                    for non-compliant resources using AWS Lambda or AWS Config rules to
+                    enforce compliance.
+                </li>
+
+                <li>
+                    <strong>Continuous Monitoring:</strong> Regularly monitor your
+                    environment with tools like AWS Config, CloudWatch, and GuardDuty to
+                    ensure ongoing compliance with CIS benchmarks.
+                </li>
+            </ul>
+            <h4>7. Documentation and Reporting</h4>
+            <ul>
+                <li>
+                    <strong>Compliance Documentation:</strong> Maintain documentation of
+                    your CIS compliance efforts, including configurations, policies, and
+                    monitoring setups.
+                </li>
+
+                <li>
+                    <strong>Regular Audits:</strong> Schedule regular internal audits to
+                    review your configurations against the CIS benchmarks and update them as
+                    needed.
+                </li>
```

Comment:
Is there a reason we didn't add this as well like what we had on ISO?

By following these steps, you'll align your AWS cloud infrastructure with ISO 27001 requirements and help ensure the security of your information assets. Remember that achieving ISO 27001 compliance is an ongoing process, requiring regular reviews and updates to your ISMS.